css-scrollbars-1/Overview.bs

```diff
@@ -230,6 +257,13 @@ authors should use ''overflow: hidden'' instead of ''scrollbar-width: none''.
 </dd>
 </dl>
 
+<div class=note>Note: Users who find the ''thin'' style of scrollbars unusable
+can include the following rule in their [=user style sheet=]:
+<pre><code class=lang-css>* { scrollbar-width: auto !important; }</code></pre>
```

Comment:
We should probably recommend UAs a way to ignore `scrollbar-width: thin` instead, as this would also remove the effect of `scrollbar-width: none`, which might be undesirable.